waspc/run

```diff
@@ -114,6 +115,9 @@ case $COMMAND in
             echo_and_eval "$TEST_UNIT_CMD --test-options \"-p \\\"$TEST_PATTERN\\\"\""
         fi
 
+        ;;
+    test:cli)
```

Comment:
You will also want to add `test-cli` above in the "usage" output of the command, so people can learn about it.

Reply:
Good catch, thanks will do